tests/e2e-evm/test/extensions/viem.ts

```diff
@@ -49,4 +50,18 @@ export function extendViem(hre: HardhatRuntimeEnvironment) {
 
   hre.viem.getWalletClient = (address: Address, walletClientConfig?: Partial<WalletClientConfig>) =>
     getWalletClient(address, { ...walletClientConfig, ...getChainConfig(hre) });
+
+  hre.viem.deployContract = (async <CN extends string>(
+    contractName: ContractName<CN>,
+    // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+    constructorArgs: any[] = [],
```

Comment:
`any` is used in the upstream type, so linter ignore was added instead of correcting it